tests/server-multiple-retention-policies/config/poll-members.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
I like this 👍 , a lot better than other tests, where we have to do this manually 😄